answerking_app/mixins/OrderMixins.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.request import Request
+from rest_framework.utils.serializer_helpers import ReturnDict
+
+from answerking_app.models.models import Order
+from answerking_app.models.serializers import OrderSerializer
+from answerking_app.views.ErrorType import ErrorMessage
+
+
+class OrderUpdateMixin:
+    def update(self, request: Request, order_id: str, *args, **kwargs) -> Response:
+        order: Order | None = Order.objects.get(pk=order_id)
+        if not order:
+            error_msg: ErrorMessage = {
+                "error": {
+                    "message": "Request failed",
+                    "details": "Object not found",
+                }
+            }
+            return Response(
+                error_msg,
+                status=status.HTTP_404_NOT_FOUND,
+            )
+
+        serializer = OrderSerializer(order, data=request.data, partial=True)
+        serializer.is_valid(raise_exception=True)
+
+        if not serializer:
```

Comment:
s this if statement  needed? I don't think so as we have run serializer.is_valid() so any errors will be raised. So I'm not sure what this is checking?

Reply:
This file is no longer needed